src/playground/addons/story-utils/index.js

```diff
@@ -525,14 +525,97 @@ export const getLinkKnobs = (data) => {
   return data;
 };
 
-export const getComplianceKnob = (data) => {
-  data['_compliance_'] = boolean(
-    '_compliance_',
-    data['_compliance_'],
-    tabLabels.checks
-  );
-
-  return data;
+export const getFormControls = (data) => {
+  return {
+    label: {
```

Comment:
Thanks for the utils !! 

I updated checkbox accordingly, see: https://github.com/ec-europa/europa-component-library/pull/1834
You can now maybe use  `getFormControls(data, 'element')` in this case.

I also removed the ending dots on the description since we have never use them on existing migrated components.